### PR TITLE
Exclude tests and add license file to pip package

### PR DESCRIPTION
**Pull request by roederja2**
Adding the XML tests files so the tests can be run from a conda build

### CODE REVIEW

**Comment by roederja2** [comment]
Actually thinking about it - we probably want to either exclude the tests module or move it inside the msal module as it currently gets installed as a top level module called "tests" which is probably not what we want.

File: setup.py [hunk 1]
```diff
@@ -67,6 +67,8 @@
         'Operating System :: OS Independent',
     ],
     packages=find_packages(),
+    package_data={'tests': '*.xml'},
```
Comment (rayluo) [inline setup.py]: Actually we intended to do the same as you later described "we probably want to either exclude the tests module". So the way to do it, is to modify the previous line to be `packages=find_packages(exclude=["tests"]),`. If you would kindly make that change and roll back this line, we would happily accept it!

File: setup.py [hunk 2]
```diff
@@ -67,6 +67,8 @@
...
+    data_files=[('', ['LICENSE'])],
```
Comment (rayluo) [inline setup.py]: I haven't personally used conda. Does msal in conda still ship with an `PKG-INFO` file which would then contain a line as `License: MIT`, just like what pip did? If so, I would suggest to also rollback this line too.

Comment (roederja2) [reply setup.py]: Conda Forge is quite keen on actually having an actual license_file: https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

**Pull request merged**